apps/web/components/Card.tsx

```diff
@@ -0,0 +1,18 @@
+import { TemplateInterface } from "../utils/getAllTemplates";
+
+interface CardProps {
+  template: TemplateInterface;
+}
+
+export default function Card({ template }: CardProps) {
+  return (
+    <a href={template.link} className="max-w-xs xl:max-w-none">
+      <img
+        className="rounded-lg"
+        src="/previews/digital-agency.png"
```

Comment:
Harcoded src here

Reply:
yes, pretty much all the images are hard coded. I need to take a screenshot of all the templates before making it dynamic.

Reply:
I've moved all the hard coded src to the json file. I makes the component dynamic at least. We can update the json file later on.